test-proxy/src/main/java/com/google/cloud/bigtable/testproxy/CbtTestProxy.java

```diff
@@ -146,51 +147,32 @@ public static CbtTestProxy createEncrypted(
    */
   private static BigtableDataSettings.Builder overrideTimeoutSetting(
       Duration newTimeout, BigtableDataSettings.Builder settingsBuilder) {
-    // TODO(developer): remove the initialRpcTimeout update below by updating the client library.
-    Duration initialRpcTimeout =
-        settingsBuilder
-            .stubSettings()
-            .bulkMutateRowsSettings()
-            .getRetrySettings()
-            .getInitialRpcTimeout();
-    if (initialRpcTimeout.compareTo(newTimeout) > 0) {
-      // Total timeout is smaller than initialRpcTimeout, which will cause deadline-related problem.
-      initialRpcTimeout = newTimeout;
-    }
-    settingsBuilder
-        .stubSettings()
-        .bulkMutateRowsSettings()
-        .retrySettings()
-        .setTotalTimeout(newTimeout)
-        .setInitialRpcTimeout(initialRpcTimeout);
-
-    settingsBuilder.stubSettings().mutateRowSettings().retrySettings().setTotalTimeout(newTimeout);
-
-    settingsBuilder.stubSettings().readRowSettings().retrySettings().setTotalTimeout(newTimeout);
-
-    settingsBuilder.stubSettings().readRowsSettings().retrySettings().setTotalTimeout(newTimeout);
-
-    settingsBuilder
-        .stubSettings()
-        .sampleRowKeysSettings()
```

Comment:
should sampleRowKeysSettings also be updated like the other methods?

Reply:
oops, yes!
good catch!